<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimMainWindow</class>
 <widget class="QMainWindow" name="SimMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point-based Soft Body Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="SimGlWidget" name="glWidgetWorld" native="true">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="SimGlWidget" name="glWidgetRest" native="true"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Particle Info:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MatrixViewer" name="matrixViewer" native="true">
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="statusTip">
    <string>Status</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlayPause"/>
   <addaction name="actionNext"/>
  </widget>
  <action name="actionNext">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/resources/step-forward-normal.png</normaloff>:/images/resources/step-forward-normal.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Take a single timestep</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="actionPlayPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/resources/pause-normal.png</normaloff>
     <normalon>:/images/resources/play-normal.png</normalon>:/images/resources/pause-normal.png</iconset>
   </property>
   <property name="text">
    <string>Play/Pause</string>
   </property>
   <property name="toolTip">
    <string>Play/Pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatrixViewer</class>
   <extends>QWidget</extends>
   <header>MatrixViewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimGlWidget</class>
   <extends>QWidget</extends>
   <header>SimGlWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
